yb-voyager/src/transform/transform.go

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
does it make sense to move this under `query` folder?

Reply:
Lets do that in further PRs if we feel the need.

Also do you mean to move it just to query folder or under queryparser package.
I agree with first one.

Also we should probably name it sql/sqlparser instead of query(which is only DMLs) but we deal with DDLs also

Reply:
done. Also renamed the package as discussed